scripts/tracecontext-integration-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

I think `/bin/bash` is usually safe, but better to use `env` in case the user wants a different `bash`.

Reply:
AFAIK only /bin/sh is safe (sh is guaranteed by POSIX to exist)

Reply:
happy to take lowest common denominator to reduced potential issues. I'll go with /bin/sh.